src/components/main.tsx

```diff
@@ -18,25 +22,60 @@ export function Main(): JSX.Element {
   }
 
   return (
-    <div className="text-center p-2 h-screen">
-      <div className="max-w-6xl text-left mx-auto ">
-        <div className="flex justify-between">
-          <DfxLogo />
-          <Connect />
+    <>
+      {/* MODALS */}
+      <StyledModal
+        type={StyledModalTypes.ALERT}
+        isVisible={needsSignUp}
+        onClose={() => {
+          // intentionally do nothing, as alert are closed via button
+        }}
+      >
+        <div className="flex flex-col">
+          <h1>Terms and Conditions.</h1>
+          <p>
+            Please read our terms and conditions and click on ”Next” to confirm and to continue to the DFX Multichain
+            Service.
+          </p>
+          <a
+            className="underline underline-offset-2 pt-6 pb-4"
+            target="_blank"
+            href={process.env.REACT_APP_TNC_URL}
+            rel="noopener noreferrer"
+          >
+            Terms and conditions DFX Swiss.
+          </a>
         </div>
-        <div className="md:flex justify-between mt-6">
-          <div className="basis-3/5 max-w-[50%] px-6 mx-auto md:mx-0">
-            <DfxTitleSection heading="DFX Multichain" subheading="Buy • Sell • Convert • Stake" />
+
+        <StyledButton
+          width={StyledButtonWidths.MD}
+          color={StyledButtonColors.RED}
+          label="Next"
+          caps={false}
+          onClick={() => signUp()}
+        />
+      </StyledModal>
```

Comment:
Can we move this to the `Connect` component? It then would be in one place.

Reply:
thought about it and there are arguments to have it in either of those. I am fine with both